test/metabase/api/card_test.clj

```diff
@@ -402,6 +401,43 @@
                (into #{} (map :name) (mt/user-http-request :rasta :get 200 "card"
                                                            :f :using_model :model_id model-id))))))))
 
+(deftest get-cards-with-last-edit-info-test
+  (mt/with-temp [:model/Card {card-1-id :id} {:name "Card 1"}
+                 :model/Card {card-2-id :id} {:name "Card 2"}]
+    (with-cards-in-readable-collection [card-1-id card-2-id]
+      (doseq [user-id [(mt/user->id :rasta) (mt/user->id :crowberto)]]
+        (revision/push-revision!
+         :entity      :model/Card
+         :id          card-1-id
+         :user-id     user-id
+         :is_creation true
+         :object      {:id card-1-id}))
+
+      (doseq [user-id [(mt/user->id :crowberto) (mt/user->id :rasta)]]
+        (revision/push-revision!
+         :entity      :model/Card
+         :id          card-2-id
+         :user-id     user-id
+         :is_creation true
+         :object      {:id card-2-id}))
+
+      (is (=? #{{:name           "Card 1"
+                 :last-edit-info {:id         (mt/user->id :rasta)
+                                  :email      "rasta@metabase.com"
+                                  :first_name "Rasta"
+                                  :last_name  "Toucan"
+                                  :timestamp  true}}
+                {:name           "Card 2"
+                 :last-edit-info {:id         (mt/user->id :crowberto)
+                                  :email      "crowberto@metabase.com"
+                                  :first_name "Crowberto"
+                                  :last_name  "Corv"
+                                  :timestamp  true}}}
+              (->> (mt/user-http-request :rasta :get 200 "card")
+                   (map #(select-keys % [:name :last-edit-info]))
+                   (map #(update-in % [:last-edit-info :timestamp] boolean))
+                   set))))))
```

Comment:
Rather than updating the result to fit the limitations of `=?`, you could simply grab the corresponding results out individually by their ID like so:
```suggestion
      (let [results (m/index-by :id (mt/user-http-request :rasta :get 200 "card"))]
        (is (=? {:name           "Card 1"
                 :last-edit-info {:id         (mt/user->id :rasta)
                                  :email      "rasta@metabase.com"
                                  :first_name "Rasta"
                                  :last_name  "Toucan"
                                  :timestamp  some?}}
                (get results card-1-id)))
        (is (=? {:name           "Card 2"
                 :last-edit-info {:id         (mt/user->id :crowberto)
                                  :email      "crowberto@metabase.com"
                                  :first_name "Crowberto"
                                  :last_name  "Corv"
                                  :timestamp  true}}
                (get results card-2-id))))))
```

Reply:
nice, did that in 345498771a